vue-frontend/src/components/home-new/BlogSection.vue

```diff
@@ -120,7 +120,17 @@ export default {
           });
 
           for (let i = 0; i < blogs.length; i++) {
-            let date = moment(String(blogs[i].pubDate)).format("MMM DD, YYYY");
+            let postDate = blogs[i].pubDate;
+            let postDay = new Date(postDate).toLocaleDateString("en-GB", {
+              month: "short",
+              day: "numeric",
+            });
+
+            let postYear = new Date(postDate).toLocaleDateString("en-GB", {
+              year: "numeric",
+            });
+            const date = postDay + ", " + postYear;
```

Comment:
Can't we use 
```
date  =  new Date(postDate).toLocaleDateString("en-GB", {
              month: "short",
              day: "numeric",
            }) + ", " + new Date(postDate).toLocaleDateString("en-GB", {
              year: "numeric",
            })
```

Reply:
i have used that but it was not showing any value so i separate this two function .